docs/dl-getting-started.md

```diff
@@ -114,6 +114,13 @@ In the contract argument, the value specified with the key `asset_id` must be un
 In Scalar DL 1.0, `asset_id` is not a reserved json key name and you can use any json key name, but this will probably be changed in future versions.
 (`asset_id` and `asset_ids` will be reserved for future enhancements.)
 
+## Create your contracts
+
+As we explained above, what you need to do is implement the `invoke` method as you like.
+We are preparing more sample contracts, so please wait for an update.
+
+To quickly run and test your contrats, [Scalar DL Emulator](https://github.com/scalar-labs/scalardl-emulator) might be useful.
+It uses mutable in-memory ledger instead of immutable ledger database and provides interactive interface, so you can do trial and errors more easily.
```

Comment:
```suggestion
It uses a mutable in-memory ledger instead of an immutable ledger database and provides an interactive interface, making it easy to do trial and error testing.
```